src/module/User/src/User/View/Helper/UserHelper.php

```diff
@@ -40,4 +41,39 @@ public function getAuthenticatedUserID() {
         $user = $this->userManager->getUserFromAuthenticator();
         return $user ? $user->getId() : '';
     }
+
+    public function getAuthenticatedUserName(){
+        $user = $this->userManager->getUserFromAuthenticator();
+        return $user ? $user->getUsername() : '';
+    }
+
+    public function isUserLoggedIn(){
+        return ($this->getAuthenticatedUserID() != '');
+    }
+
+    public function createJWSScrollback(){
+        $payload = array(
+            "iss" => "http://serlo.org",
+            "sub" => "".$this->getAuthenticatedUserEMail()."",
+            "aud" => "scrollback.io",
+            "iat" => time(),
+            "exp" => time()+60
+        );
+        $key = 'dummy_key';
```

Comment:
create an environment variable for this and document it. Also set this for testing purposes in bash_profile